show/main.py

```diff
@@ -1542,8 +1542,26 @@ def bgp(verbose):
 @click.option('--verbose', is_flag=True, help="Enable verbose output")
 def ntp(verbose):
     """Show NTP information"""
-    cmd = "ntpq -p -n"
-    run_command(cmd, display_cmd=verbose)
+    ntpcmd = "ntpq -p -n"
+    if ctx.invoked_subcommand is None:
+        cmd = 'sonic-cfggen -d --var-json "MGMT_VRF_CONFIG"'
+
+        p = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, stderr=subb
+process.PIPE)
```

Comment:
Improper line break. Also, the "subprocess" which is broken mid-word is misspelled ('subbprocess'). Was this tested?

Reply:
Something has gone wrong in checkin process. Will check and update.

Reply:
@jleveque: Thanks for pointing it out. Corrected the mistake. Something went wrong while copying the file.